config/more_samples-glyf_colr_1.py

```diff
@@ -269,7 +288,75 @@ def _paint_scale(scale_x, scale_y, center_x, center_y, accessor_char):
         glyph_name=glyph_name,
         accessor=accessor_char,
         advance=_UPEM,
-        glyph=pen.glyph(),
+        glyph=_upem_box_pen().glyph(),
+        colr=colr,
+    )
+
+
+def _extend_modes(gradient_format, extend_mode, accessor_char):
+
+    format_map = {
+        "linear": ot.PaintFormat.PaintLinearGradient,
+        "radial": ot.PaintFormat.PaintRadialGradient,
+    }
+
+    if gradient_format not in format_map.keys():
```

Comment:
don't need `.keys()` to check containment in a dict
```suggestion
    if gradient_format not in format_map:
```

Reply:
Done in two cases, thanks for the suggestion.